### ISSUE TITLE
npm install fails with the following error

### ISSUE THREAD

**Issue by asuraphel** [enketo/enketo-transformer#48]
My machine is running Ubuntu 18.04 with npm version 6.4.1. The error is as follows: 

> 79 verbose stack Error: libxslt@0.6.5 install: `node-gyp rebuild`
> 79 verbose stack Exit status 1
> 79 verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:301:16)
> 79 verbose stack     at emitTwo (events.js:125:13)
> 79 verbose stack     at EventEmitter.emit (events.js:213:7)
> 79 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
> 79 verbose stack     at emitTwo (events.js:125:13)
> 79 verbose stack     at ChildProcess.emit (events.js:213:7)
> 79 verbose stack     at maybeClose (internal/child_process.js:897:16)
> 79 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:208:5)
> 80 verbose pkgid libxslt@0.6.5
> 81 verbose cwd /home/alexander/playground/enketo-transformer
> 82 verbose Linux 4.15.0-36-generic
> 83 verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--unsafe-perm"
> 84 verbose node v8.1.3
> 85 verbose npm  v6.4.1
> 86 error code ELIFECYCLE
> 87 error errno 1
> 88 error libxslt@0.6.5 install: `node-gyp rebuild`
> 88 error Exit status 1
> 89 error Failed at the libxslt@0.6.5 install script.
> 89 error This is probably not a problem with npm. There is likely additional logging output above.
> 90 verbose exit [ 1, true ]

**Comment by MartijnR** [comment]
I'm not aware of an issue with libxslt@0.6.5 with that node and npm version. I just tried with node 8.11.2 (slightly newer than yours) and npm 6.4.1 on Mac OS. 

However, I have noticed that these native modules (libxslt and also node-sass) fail to install occasionally for unknown reasons. I usually do a `rm -R node_modules` and then run `npm install` again and that usually works.

If the issue persists, I'd do:
1. create a test folder
2. run `npm install libxslt@0.6.5` in that folder
3. if you see the same problem, and could post an issue here: https://github.com/albanm/node-libxslt/issues
